README.md

```diff
@@ -15,7 +36,17 @@ An optical data set can include metadata like the [standard](https://github.com/
 
 Similarly, [calorimetric.json](https://github.com/ise621/building-envelope-data/blob/develop/schemas/calorimetric.json) defines the metadata about a calorimetric data set and refers to [calorimetricData.json](https://github.com/ise621/building-envelope-data/blob/develop/schemas/calorimetricData.json) for the pure calorimetric data which is illustrated by the example [bistMeasurement.json](https://github.com/ise621/building-envelope-data/blob/develop/examples/dbe/calorimetric/bistMeasurement.json).
 
-## On your Linux machine
+## How to use this repository
+
+### For beginners
+
+With you web browser, you can search our [wiki](https://github.com/ise621/building-envelope-data/wiki), the [issues](https://github.com/ise621/building-envelope-data/issues) and [pull requests](https://github.com/ise621/building-envelope-data/pulls) and contribute to them.
+
+In order to browse the code conveniently, you should first [clone the repository](https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/cloning-a-repository) and then use a text editor, for example [Visual Studio Code](https://code.visualstudio.com/).
+
+### On your Linux machine
+
+In order to use all functions of this code for example to format and test, please follow the instructions below.
```

Comment:
Rephrase to: `In order to use our development tooling, for example, to format code and to run tests, follow the instructions below.`